src/main/java/io/confluent/examples/streams/microservices/AddInventory.java

```diff
@@ -0,0 +1,86 @@
+package io.confluent.examples.streams.microservices;
+
+import io.confluent.examples.streams.avro.microservices.OrderState;
+import io.confluent.examples.streams.avro.microservices.Product;
+import io.confluent.examples.streams.microservices.domain.Schemas;
+import io.confluent.examples.streams.microservices.domain.Schemas.Topics;
+import io.confluent.examples.streams.microservices.domain.beans.OrderBean;
+import io.confluent.examples.streams.microservices.util.Paths;
+import io.confluent.examples.streams.utils.MonitoringInterceptorUtils;
+
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.streams.StreamsConfig;
+
+import org.apache.kafka.common.serialization.Serdes;
+
+import org.glassfish.jersey.client.ClientConfig;
```

Comment:
There are some unused imports in this class.

Reply:
@vvcephei : I can't get `checkstyle` working to identify the unused imports in this class.  Do you mind telling which ones they are and then I'll remove them?